lepton/image.go

```diff
@@ -359,8 +359,7 @@ func DownloadReleaseImages(version string) error {
 }
 
 func DownloadFile(filepath string, url string, timeout int) error {
-
-	fmt.Println("Downloading..", filepath)
+	fmt.Println("Downloading..", url)
```

Comment:
Quick fix. It now says "Downloading.. <url>" instead of "Downloading.. <$OPS_HOME>"